CDP4Orm/Dao/Resolve/ContainerDao.cs

```diff
@@ -207,18 +216,20 @@ public class ContainerDao : IContainerDao
                 otherPartition = partition.Replace(Utils.IterationSubPartition, Utils.EngineeringModelPartition);
             }
 
+            var dao = this.DaoResolver.QueryDaoByTypeName(typeName);
+            
             sqlBuilder.Append(
```

Comment:
may i request we refactor this so it becomes more readable. using the numbered arguments and a combi of string builder and string.format can be improved

Reply:
refactored